CONTRIBUTING.md

```diff
@@ -113,10 +132,10 @@ If you are interested in creating a translation, here are some pointers for how
 * Often there is work to do in creating markdown files or updating the translation to keep it up to date with latest changes.
 * We would request that you base your translation on the 4.0/en folder in the master branch as this is now static at the 4.0.3 version.
 * In order to start a translation, please start by forking the ASVS repository.
-    * If you are updating an existing translation which has markdown (just French as at December 2021), you can make modifications to the files in the existing folder based on language code (just /fr as at December 2021).
-    * if you are starting a new markdown translation, take a copy of the /en folder and rename it to the 2 character language code which will be used for the translation. 
+    * If you are updating an existing translation which has markdown (just French as of December 2021), you can make modifications to the files in the existing folder based on language code (just /fr as at December 2021).
+    * If you are starting a new markdown translation, take a copy of the /en folder and rename it to the 2 character language code which will be used for the translation.
```

Comment:
Smal edit

    * If you are starting a new markdown translation, take a copy of the /en folder and rename it to the 2-character language code used for the translation.

Reply:
Done